backend/routes/shifts.ts

```diff
@@ -38,8 +38,8 @@ router.post('/', async (req: any, res: any) => {
 
   try {
     shift = await shift.save();
-    console.log(shift);
-    res.json(shift);
+    const final = await shift.populate('event');
+    res.json(final);
```

Comment:
I added the `populate` function here because the event title and date were showing up as undefined / error in the `pastShifts` page when navigating to it after logging a shift